README.md

```diff
@@ -1,2 +1,31 @@
-# jbot
-Red Hat Jira Bot
+# JIRA Bot 👋
+
+A jira bot which fetches JIRA issues for you and a bunch of other stuff
```

Comment:
A bot that retrieves the JIRA issue details from your JIRA server.

Reply:
Changed to 
```
A bot that retrieves the JIRA issue details from your JIRA server.
```